google-cloud-testing/storage-benchwrapper/README.md

```diff
@@ -0,0 +1,67 @@
+# storage-benchwrapper
+
+storage-benchwrapper is a gRPC wrapper around the storage library for benchmarking purposes.
+
+## Running
+
+```
+mvn clean install -DskipTests=true (one time)
+cd google-cloud-testing/storage-benchwrapper
+export STORAGE_EMULATOR_HOST=localhost:8080
+mvn clean install exec:java -DskipTests=true -Dport=8081
```

Comment:
`exec:java` doesn't actually fork to a subprocess and as such doesn't properly handle send JVM level shutdown signals. If you are depending on JVM level shutdown signals (such as shutting down daemon thread pools) you'll want to switch to using `exec:exec`.

Here is a PR where I did just that if you want to use it as a starting place: https://github.com/GoogleCloudPlatform/java-docs-samples/pull/1523

Reply:
Also, why clean and install? exec:exec or exec:java should do it as needed.

Reply:
> exec:java doesn't actually fork to a subprocess and as such doesn't properly handle send JVM level shutdown signals.

Thanks. I'm not, so I'll save myself `pom.xml` frustration and skip that suggestion. :)

> Also, why clean and install? exec:exec or exec:java should do it as needed.

Doesn't seem to: https://gist.github.com/jadekler/d88583bc1c221ef9f1adb44ee88d6d0d